benchmarks/apps/sombrero/sombrero.py

```diff
@@ -122,4 +137,7 @@ def set_up_from_parameters(self):
 
     @run_after('setup')
     def setup_num_tasks(self):
-        self.num_tasks = self.current_partition.processor.num_cores * 64
+        self.num_tasks = max_num_tasks(
+            self.current_partition.processor.num_cores * 64,
```

Comment:
I'm not sure whether `64` should multiply `self.current_partition.processor.num_cores` or the result of `max_num_tasks`.  In the latter case we'd greatly oversubscribe the node, right?  But that was already the case before, so probably that was it?

Reply:
> In the latter case we'd greatly oversubscribe the node, right? But that was already the case before, so probably that was it?

I think this test was intended to run on 64 nodes so it shouldn't be oversubscribed. Maybe we should use the `num_tasks_per_node` option instead and calculate the `max_num_tasks` according to `LATTICE_VOLUME / 64` to ensure even load balancing and that we actually use 64 nodes?